services/jobrunner/playbook.md

```diff
@@ -67,23 +67,26 @@ If you wish to change the config in `01_defaults.env` or `03_backend.env`, you
 need to merge a change to the `services/jobrunner/defaults.env` or
 `BACKEND/backend.env`, and update the infrastructure code as above.
 
-### Update job-runner
-
-#### Backend-server / just
+### Deploy job-runner
 
 In a clone of the [backend-server](https://github.com/opensafely-core/backend-server) repository, run:
 
 ```bash
 just update-jobrunner
 ```
 
-#### Manual
+If there are dependency updates cd into `/home/jobrunner/jobrunner/lib` and run:
+
+    git pull
+
 
-In `/home/jobrunner/jobrunner/code` run:
+If there are config updates cd into `/home/jobrunner/jobrunner/config` and run:
```

Comment:
the contents of `/home/jobrunner/jobrunner/config` are managed by https://github.com/opensafely-core/backend-server/blob/f3918ab5210ef303e13995f9f794cdf6f64d01d9/services/jobrunner/install.sh#L46-L59 - currently to bring them in you'd need to run `just manage` to re-run the whole install (which should work, but...)

Reply:
actually, I was planning to pull some of that stuff out of the jobrunner install script & into its own script - as I think this config is used for other services too - but haven't got there yet

Reply:
Oh! I think the playbook has changed since I last looked at these 😂  it's a bit weird that files 1&3 in that directory shouldn't be edited 😬

Reply:
yeah, it's a bit weird, we're working on it...